Volcano/README.md

```diff
@@ -0,0 +1,89 @@
+### c) Copy and paste
+Navigate to a section of a notebook you are interested in and copy-paste the code you want to use into a notebook on your computer.
+
+### 3. Install the environment file
+Create an environment called `geonet-tutorial` from the specifications in the file `environment.yml`.
+
+`conda env create -f environment.yml`
+
+And then activate this environment.
+
+`conda activate geonet-tutorial`
+
+Install the Python kernel in this environment.
+
+`conda install -c conda-forge ipykernel`
+
+`python -m ipykernel install --user --name=geonet-tutorial`
+
+We recommend using [JupyterLab](https://jupyterlab.readthedocs.io/en/stable/index.html) rather than the older Jupyter Notebook.
+
+If you don't have JupyterLab installed, here are [detailed instructions](https://jupyterlab.readthedocs.io/en/stable/getting_started/installation.html).
+
+Open JupyterLab
+
+From the Linux command line
+
+`jupyter lab`
+
+### 4. Running tutorials as standalone scripts
+Tutorials are only available as Jupyter notebooks. If you want to run a notebook as a standalone Python script you can do that. Open the notebook in JupyterLab and export it as an [Executable Script](https://jupyterlab.readthedocs.io/en/stable/user/export.html).
+                                                      
+## Data Blogs
+
+Data blogs are news stories focussed on GeoNet data and how to use
+and understand it. They were first published in June 2022 and are accessible through the [GeoNet News web page](https://www.geonet.org.nz/news). While data blogs are not tutorials, the material sometimes contains
+code excerpts and examples that our data users will find helpful, such as Jupyter notebooks and shell scripts. The material in the repository is that used by the blog's authors to prepare the blog at the time it was written. In contrast to data tutorials, with blogs we make no effort to review and keep up to date Jupyter notebooks, shell scripts, or any other code-like material. Also, we do not provide the environment and software versions that we used in preparing blog material. In many cases, the Python environment in [environment.yml](./environment.yml) may work if you want to run a Jupyter notebook used to generate material for a blog. If you are having difficulties, please ask us.
+
+Code from data blogs are stored in the folder [Data_Blog](./Data_Blog). Within this is a
+sequentially numbered folder including a shortened version of the blog's subject. For example, the folder *blog_01_val* contains blog number 01, and the subject was Volcanic Alert Level (VAL) data. Not all blogs have code so not all folders are in the repository.
+
+If you want an up to date list of published blogs, go to the [News section on our web page](https://www.geonet.org.nz/news), filter for Data Blog and then hit the Search button.
```

Comment:
this is an exact copy of the main README in home github repo. I'm sure that's unintentional, so can you please check if you have a "volcano data tutorials" specific README?

Reply:
I don't know how that happened. Yes, there is a specific README for the volcano folder.